buildenv/jenkins/build

```diff
@@ -0,0 +1,229 @@
+                } catch(ex) {
+                    OPENJ9_BRANCH_OPTION = ''
+                }
+                try {
+                    if ( OPENJ9_SHA != "" ) {
+                        OPENJ9_SHA_OPTION = "-openj9-sha=${OPENJ9_SHA}"
+                    } else {
+                        OPENJ9_SHA_OPTION = ''
+                    }
+                } catch (ex) {
+                    OPENJ9_SHA_OPTION = ''
+                }
+                try {
+                    if ( OMR_REPO != "" ) {
+                        OMR_REPO_OPTION = "-omr-repo=${OMR_REPO}"
+                    } else {
+                        OMR_REPO_OPTION = ''
+                    }
+                } catch(ex) {
+                    OMR_REPO_OPTION = ''
+                }
+                try {
+                    if ( OMR_BRANCH != "" ) {
+                        OMR_BRANCH_OPTION = "-omr-branch=${OMR_BRANCH}"
+                    } else {
+                        OMR_BRANCH_OPTION = ''
+                    }
+                } catch(ex) {
+                    OMR_BRANCH_OPTION = ''
+                }
+                try {
+                    if ( OMR_SHA != "" ) {
+                        OMR_SHA_OPTION = "-omr-sha=${OMR_SHA}"
+                    } else {
+                        OMR_SHA_OPTION = ''
+                    }
+                } catch(ex) {
+                    OMR_SHA_OPTION = ''
+                }
+            }
+
+            git branch: "${OPENJDK_BRANCH}", url: "${OPENJDK_REPO}"
+            sh "git checkout ${OPENJDK_SHA}"
+            sh "bash ./get_source.sh ${OPENJ9_REPO_OPTION} ${OPENJ9_BRANCH_OPTION} ${OPENJ9_SHA_OPTION} ${OMR_REPO_OPTION} ${OMR_BRANCH_OPTION} ${OMR_SHA_OPTION}"
+        }
+    }
+}
+
+def checkout_pullrequest() {
+    stage('Checkout Pull Request') {
+        timestamps {
+            echo 'Look for Dependent Changes'
+            script {
+                def KEYWORD = "depends"
+                def COMMENT = ghprbCommentBody.toLowerCase(Locale.US);
+                def OJDK_PR
+                def OMR_PR
+                omr_bool     = false
+                omr_upstream = false
+                openjdk_bool = false
+
+                if (COMMENT.contains(KEYWORD)) {
+
+                    // Gather everything after KEYWORD. Assumes a whitespace after KEYWORD, hence +1
+                    def DEPENDS_BLOB  = COMMENT.substring(COMMENT.indexOf(KEYWORD) + KEYWORD.length() + 1 , COMMENT.length())
+                    def DEPENDS_ARRAY = DEPENDS_BLOB.tokenize(' ')
+
+
+                    for (DEPEND in DEPENDS_ARRAY) {
+                        String REPO  = DEPEND.substring(DEPEND.indexOf("/") + 1, DEPEND.indexOf("#"));
+                        String PR_ID = DEPEND.substring(DEPEND.indexOf("#") + 1, DEPEND.length());
+                        switch(REPO) {
+                            case "omr":
+                                if (!omr_bool) {
+                                    omr_bool     = true
```

Comment:
smae here

Reply:
Rather than pointing out each occurrence, I'll stop mentioning this and expect you can catch the remaining locations.